environment-python_and_r.yml

```diff
@@ -18,12 +18,10 @@ dependencies:
   - geolinks
   - geopandas
   - gridgeo
+  - hvplot
   - ioos-tools
   - jupyter
-  - jupyter_contrib_nbextensions
-  - jupyterthemes
-  - matplotlib=2
-  - nb_conda_kernels
```

Comment:
`jupyter_contrib_nbextensions` and `nb_conda_kernels` are causing problems on Windows so I removed them. They are not essential.